-----------Exception Details on  2020-10-04 13:45:37-----------------
-------------------------------------------------------------------------------------
Log Written Date: 2020-10-04 13:45:37
Error Line No:    at System.IO.FileStream.ValidateFileHandle(SafeFileHandle fileHandle)
   at System.IO.FileStream.CreateFileOpenHandle(FileMode mode, FileShare share, FileOptions options)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options)
   at System.IO.FileStream..ctor(String path, FileMode mode)
   at SCustomers.Services.TransferService.UploadCardAsync(CardUploadDto request, CancellationToken cancellationToken) in C:\Users\Moses Mckinkela\source\repos\SCustomers\SCustomers\Services\TransferService.cs:line 647
   at SCustomers.Controllers.TransfersController.OnPostCardImage(CardUploadDto request, CancellationToken cancellationToken) in C:\Users\Moses Mckinkela\source\repos\SCustomers\SCustomers\Controllers\TransfersController.cs:line 44
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextExceptionFilterAsync>g__Awaited|25_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
Error Type: System.IO.DirectoryNotFoundException
Error Message: Could not find a part of the path 'C:\Users\Moses Mckinkela\source\repos\SCustomers\SCustomers\wwwroot\uploads\Screenshot (2).png'.
InnerError Line No: 
InnerError Type: 
InnerError Message: 
--------------------------------*End*------------------------------------------



